src/workerd/io/io-context.c++

```diff
@@ -1182,6 +1197,10 @@ bool IoContext::hasCurrent() {
   return threadLocalRequest != nullptr;
 }
 
+bool IoContext::isCurrentNull() {
+  return threadLocalRequest == nullptr;
+}
```

Comment:
There is already an `IoContext::hasCurrent()` that this duplicates

Reply:
I actually saw `hasCurrent` and then still wrote this function for some reason :facepalm: